bucket/tfsec.json

```
@@ -0,0 +1,26 @@
+{
+    "version": "0.47.0",
+    "description": "tfsec uses static analysis of your terraform templates to spot potential security issues. Now with terraform v0.12+ support.",
```

The `tfsec` strapline keeps on changing so it might be best to remove the dynamic part.

```suggestion
    "description": "tfsec uses static analysis of your terraform templates to spot potential security issues.",
```

---

changes made in https://github.com/ScoopInstaller/Main/pull/2680